build/BuildDefaults.props

```diff
@@ -2,11 +2,12 @@
   <PropertyGroup>
     <CLITargets Condition=" '$(CLITargets)' == '' ">Prepare;Compile;Test;Package;Publish</CLITargets>
     <Configuration Condition=" '$(Configuration)' == '' ">Debug</Configuration>
-    <HasAdditionalSharedFramework Condition="'$(Rid)' != 'linux-x64'">true</HasAdditionalSharedFramework>
+    <HasAdditionalSharedFramework Condition=" '$(HasAdditionalSharedFramework)' == '' ">false</HasAdditionalSharedFramework>
     <IncludeAdditionalSharedFrameworks Condition=" '$(IncludeAdditionalSharedFrameworks)' == '' AND '$(HasAdditionalSharedFramework)' == 'true' ">true</IncludeAdditionalSharedFrameworks>
     <IncludeAdditionalSharedFrameworks Condition=" '$(IncludeAdditionalSharedFrameworks)' == '' ">false</IncludeAdditionalSharedFrameworks>
     <IncludeNuGetPackageArchive Condition=" '$(IncludeNuGetPackageArchive)' == '' ">true</IncludeNuGetPackageArchive>
     <SkipBuildingInstallers Condition=" '$(SkipBuildingInstallers)' == '' ">false</SkipBuildingInstallers>
     <UsePortableLinuxSharedFramework Condition=" '$(UsePortableLinuxSharedFramework)' == '' AND '$(OSPlatform)' == 'linux' ">true</UsePortableLinuxSharedFramework>
+    <IncludeLegacySharedFramework Condition=" '$(Rid)' != 'linux-x64' ">true</IncludeLegacySharedFramework>
```

Comment:
I left the additional framework mechanism in place in case we want to ship a CLI with additional runtimes again in the future, like 2.0 and 2.1.

The legacy one toggles whether we should setup the additional stage2 with legacy runtimes available to it. Turning it on won't put this runtime in the actual stage2 that gets packaged and shipped and that most of our tests use.